### ISSUE TITLE
Docker version in go.mod causing unknown revision

### ISSUE THREAD

**Issue by jpreese** [deislabs/oras#125]
Using

```bash
$ go version
go version go1.13 windows/amd64
```

With environment

```bash
$ go env
set GO111MODULE=
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\jreese\AppData\Local\go-build
set GOENV=C:\Users\jreese\AppData\Roaming\go\env
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=C:\Users\jreese\go
set GOPRIVATE=
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=c:\go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=c:\go\pkg\tool\windows_amd64
set GCCGO=gccgo
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=C:\git\github\conftest\go.mod
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=C:\Users\jreese\AppData\Local\Temp\go-build006458062=/tmp/go-build -gno-record-gcc-switches
```

I'm getting 

```bash
go: github.com/deislabs/oras@v0.7.0 requires
        github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

When trying to include deislabs/oras into my project

If I specifically use `replace` as this project has done:

```bash
replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
```

The project does build, but it seems smelly that I have to include that.

Is this an issue with oras that can be resolved, or just wonkyness with the current modules ecosystem in Go?

**Comment by jpreese** [comment]
Just popping in to see if theres any input on this!

**Comment by shizhMSFT** [comment]
Fixed by #132
